packages/advanced-logic/test/extensions/payment-context-exchange-rate.test.ts

```diff
@@ -0,0 +1,37 @@
+//import { ExtensionTypes, RequestLogicTypes } from '@requestnetwork/types';
+//import Utils from '@requestnetwork/utils';
+import 'mocha';
+
+//import erc20ProxyContract from '../../src/extensions/payment-network/erc20/proxy-contract';
```

Comment:
I don't think these commented imports will be useful on the future, you can remove them.